packages/itwin/tree-widget/.eslintrc.js

```diff
@@ -6,12 +6,15 @@ require("@rushstack/eslint-patch/modern-module-resolution");
 
 module.exports = {
   plugins: ["@itwin"],
-  extends: "plugin:@itwin/ui",
+  extends: ["plugin:@itwin/ui", "plugin:react/jsx-runtime"],
```

Comment:
will this cause issues for apps? (i have no idea, but im in favor of removing the react import if possible)

Reply:
I don't think this should cause any issues for apps. This is [React 17 feature](https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html).

Yes this eslint config change is needed to disable eslint rules that require `React` to be imported.